gtsfm/runner/run_scene_optimizer_synthetic_tanks_and_temples.py

```diff
@@ -0,0 +1,62 @@
+"""Runs GTSfM on a Tanks and Temples dataset, using synthetic correspondences.
+
+Author: John Lambert
+"""
+
+import argparse
+
+import gtsfm.utils.logger as logger_utils
+from gtsfm.loader.tanks_and_temples_loader import TanksAndTemplesLoader
+from gtsfm.loader.loader_base import LoaderBase
+from gtsfm.runner.gtsfm_runner_base import GtsfmRunnerBase
+
+logger = logger_utils.get_logger()
+
+
+_TANKS_AND_TEMPLES_RESOLUTION_PX = 1080
+
+
+class GtsfmRunnerSyntheticTanksAndTemplesLoader(GtsfmRunnerBase):
+    tag = "GTSFM with LiDAR scans, COLMAP camera poses, and image names stored in Tanks and Temples format"
```

Comment:
Why does it have to just be tanks and temples? Would this run with any dataset with a GT mesh?

Reply:
It could be for any dataset w/ a GT mesh. I've added a TODO for that.